extensions/paystack/src/hooks/paystack.ts

```diff
@@ -0,0 +1,78 @@
+import { getPreferenceValues } from '@raycast/api'
+import { useState } from 'react'
+import { PaystackResponse } from '../utils/types'
+interface Preferences {
+  liveSecretKey: string
+  testSecretKey: string
+  mode: 'live' | 'test'
+}
+
```

Comment:
```suggestion

```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`